thespian/system/transport/asyncTransportBase.py

```diff
@@ -35,9 +35,9 @@
 # transmits are immediately failed instead of being queued.
 
 MAX_PENDING_TRANSMITS = 20
-MAX_QUEUED_TRANSMITS = 950
-QUEUE_TRANSMIT_UNBLOCK_THRESHOLD = 780
-DROP_TRANSMITS_LEVEL = MAX_QUEUED_TRANSMITS + 100
+MAX_QUEUED_TRANSMITS = int(os.environ.get("THESPIAN_MAX_QUEUED_TRANSMITS", 950))
+QUEUE_TRANSMIT_UNBLOCK_THRESHOLD = int(os.environ.get("THESPIAN_QUEUED_TRANSMIT_UNBLOCK_THRESHOLD", 780))
+DROP_TRANSMITS_LEVEL = MAX_QUEUED_TRANSMITS + int(os.environ.get("THESPIAN_DROP_TRANSMITS_LEVEL", 100))
```

Comment:
Please use `getenvdef("...", int, defval)` for these as well.  If you implemented these following an example elsewhere in the code from before `getenvdef` was available, my apologies, and please feel free to add tickets for those (or make a PR).